benchmark/client/h2_client.go

```diff
@@ -247,6 +247,7 @@ func main() {
 	cfg := flag.String("cfg", "", "Configuration file")
 	log := flag.String("log", "/tmp/h2_client_log.txt", "logfile")
 	qlog_loc := flag.String("qlog", "/tmp/qlog.txt", "quiclog")
+	InsecureSkipVerify := flag.Bool("InsecureSkipVerify", false, "Disable certificate verification")
```

Comment:
flags should have underscored names.
https://google.github.io/styleguide/go/decisions.html#flags